### ISSUE TITLE
WCAG: Can we remove `user-scalable=no` from Viewport meta data?

### ISSUE THREAD

**Issue by mgesing** [smartstore/Smartstore#1333]
It violates WCAG 1.4.4 "Resize Text".

**Comment by muratcakir** [comment]
`user-scalable=no` was introduced to fix rendering problems in iOS Safari. It can be removed if the current Safari browser no longer has any problems.
